core/src/main/scala/kafka/log/Log.scala

```diff
@@ -313,10 +325,15 @@ class Log(@volatile var dir: File,
       throw new IllegalStateException("Producer state must be empty during log initialization")
     loadProducerState(logEndOffset, reloadFromCleanShutdown = hasCleanShutdownFile)
 
-    info(s"Completed load of log with ${segments.size} segments, log start offset $logStartOffset and " +
+    info(s"Completed load of log with ${segments.size} segments, local log start offset $localLogStartOffset and " +
       s"log end offset $logEndOffset in ${time.milliseconds() - startMs} ms")
   }
 
+  def updateLogStartOffsetFromRemoteTier(remoteLso: Long): Unit = {
```

Comment:
Maybe use another acronym here as LSO is already used for the Last Stable Offset?